### DISCUSSION TITLE
Express builds not working 504

### DISCUSSION THREAD

**Discussion by julio-robles** [vercel/community#3770] [Bug Report]
### Page to Investigate

Any express serverless

### Steps to Reproduce

Import a clean express js app.
Builds are not building correctly generating a 504 error.

**Comment by flaviokicis** [comment]
Same thing happening here, it started at around 3 PM GMT. We need to push a production build as soon as possible and unfortunately there is nothing we can do.

**Comment by julio-robles** [comment]
Seems like Vercel team already fixed it.
